<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrateWin</class>
 <widget class="QWidget" name="CalibrateWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalibrateWin</string>
  </property>
  <widget class="QPushButton" name="pbSetup">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLotNo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lot No</string>
   </property>
  </widget>
  <widget class="LineEdit" name="leLotNo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClear">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
